### PR TITLE
Add functions to fetch and store data

### PR DESCRIPTION
**Pull request by lwpenny**
The two functions are "fetch_data" and "store_data". fetch_data gets the data from NY Times. store_data puts the data into our GCP bucket. This is the first step in the ETL process!

### CODE REVIEW

File: fetch_data/main.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        # Instantiates a client
+        storage_client = storage.Client()
+        bucket = storage_client.get_bucket("bucket-data_raw")
+        bucket_csv = bucket.blob('data_us_nyt.csv')
+        bucket_csv.upload_from_string(data)
+        print("complete!")
+    except Exception as e:
+        print(str(e))
+
+def hello_pubsub(event, context):
```
Comment (ckingbailey) [inline fetch_data/main.py]: Can you delete this function?

**Review by ckingbailey — CHANGES_REQUESTED** [review]

File: fetch_data/main.py (hunk 1 above)
Comment (lwpenny) [reply fetch_data/main.py]: oh ok

Comment (lwpenny) [reply fetch_data/main.py]: if you insist!

Comment (lwpenny) [reply fetch_data/main.py]: I hated that function from the very beginning!

File: fetch_data/main.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import base64
+import requests
+from google.cloud import storage
+from dotenv import load_dotenv
+load_dotenv()
+import os
+
+url = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv'
+gcp_project = "test-python-etl"
```
Comment (ckingbailey) [inline fetch_data/main.py]: These variables are not used for anything, are they? Can they go away?

**Review by ckingbailey — CHANGES_REQUESTED** [review]

File: fetch_data/main.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+service_account = "service-account_local-dev"
```
Comment (ckingbailey) [inline fetch_data/main.py]: this one, too

**Review by ckingbailey — APPROVED** [review]
Looks peachy 🍑, sport 🧢

**Pull request merged**